Resources/Prototypes/Entities/Objects/Consumable/Food/breakfast.yml

```diff
@@ -0,0 +1,55 @@
+# When adding new food also add to random spawner located in Resources\Prototypes\Entities\Markers\Spawners\Random\Food_Drinks\food_breakfast.yml
+
+# Base
+
+- type: entity
+  parent: FoodInjectableBase
+  id: FoodBreakfastBase
+  abstract: true
+  description: A full breakfast, rise and shine!
+  components:
+  - type: Item
+    storedRotation: -90
+  - type: Food
+  - type: Sprite
+    sprite: Objects/Consumable/Food/breakfast.rsi
+  - type: SolutionContainerManager
+    solutions:
+      food:
+        maxVol: 40
+        reagents:
+        - ReagentId: Nutriment
+          Quantity: 25
+        - ReagentId: Vitamin
+          Quantity: 5
+  - type: Tag
+    tags:
+    - VoxFood
+    
+# breakfasts
+
+- type: entity
+  name: full american breakfast
+  parent: FoodBreakfastBase
+  id: FoodBreakfastAmerican
+  description: Dear, don’t think of getting out of bed yet.
+  components:
+  - type: FlavorProfile
+    flavors:
+      - food
+  - type: Sprite
+    state: fullamerican
+# Tastes like food.
+
+- type: entity
+  name: full english breakfast
+  parent: FoodBreakfastBase
+  id: FoodBreakfastEnglish
+  description: The closest thing you'll get to room service out here.
```

Comment:
```suggestion
  description: A huge plate of protein and carbs, more likely to put you back in bed than get you out of it. Don't mention the war!
```

Reply:
("Don't mention the war" is a reference to Fawlty Towers, a famous British sitcom about a British seaside hotel. Full English breakfasts are often served at hotel breakfasts, especially when Brits are expected to be on holiday.)